### ISSUE TITLE
Improve Wallet Element Accessibility

### ISSUE THREAD

**Issue by Douglashdaniel** [brave/brave-browser#19147]
## Description 
We need to improve on the Interactive Elements Accessibility (Buttons, Tabs, List Items, DropDowns, etc...) through out the wallet. 

Internal Feedback

None of the interactive elements (buttons, tabs, list items, dropdowns) have hover or active states. It will feel a lot more responsive with those when using a mouse.
None of those interactive elements are keyboard accessible either as there are no focus states. Everything has outline: none set, so focus position is invisible. They at least need :focus-visible states.
Solution to this is to make sure every interactive element of a design has defined a hover, pressed, and focus states (and "loading" if applicable). Developers, please ask the Designer for these if they're not there. It seems much quicker to add these as you go compared to going back and adding it in (especially when our focus has moved on)

**Comment by srirambv** [comment]
<table>
   <tr>
      <th>Brave</th>
      <th colspan="3">1.32.95 Chromium: 95.0.4638.69 (Official Build) beta (64-bit)</th>
   </tr>
   <tr>
      <th>Revision</th>
      <td colspan="3"><code>6a1600ed572fedecd573b6c2b90a22fe6392a410-refs/branch-heads/4638@{#984}</code></td>
   </tr>
   <tr>
      <th>OS
      </th>
      <th>:ballot_box_with_check: Linux</th>
      <th>:ballot_box_with_check: Windows 11 Version 21H2
         <br/> (Build 22000.282)
      </th>
      <th>:ballot_box_with_check: macOS Version 11.5.2
         <br/> (Build 20G95)
      </th>
   </tr>
   <tr>
      <td colspan=4></td>
   <tr>
      <td colspan=4>
         <ul>
         <li>Verified every element on the wallet is keyboard accessible</li>
         <li>Verified element sub selection can be done by space or by return</li>
         <li>Verified able to navigate forward and backward using keyboard</li>
        </ul>
      </td>
   </tr>
</table>
